orchagent/vnetorch.h

```diff
@@ -21,21 +21,30 @@ const request_description_t vnet_request_description = {
     { "vxlan_tunnel", "vni" } // mandatory attributes
 };
 
-enum class VNET_EXEC
```

Comment:
Why is this removed, if you want we can rename but the vnetOrch can use this to differentiate